.github/workflows/deploy.dev.yml

```diff
@@ -11,6 +11,7 @@ jobs:
     with:
       environment: dev
       app_origin: "https://www.intern.dev.nav.no"
+      decorator_url: "https://decorator-next.ekstern.dev.nav.no"
```

Comment:
`decorator_url` må også legges inn som en input i `build-and-deploy.yml`. Kan sette den til `required: false`, så den ikke trenger å settes på prod-workflow'en.